### DISCUSSION TITLE
how to reset UI layout to default?

### DISCUSSION THREAD

**Discussion by novreisb** [pawelsalawa/sqlitestudio#4826] [Q&A]
how to reset UI layout to default? 
If I delete Settings3 I lose Sql History and I don't want that. Unless I can export  this  history but i did not find how!

**Comment by pawelsalawa** [comment]
If I didn't forget anything, I think there is only one way to do it:

1. Make a copy of `settings3` file, let's call it `sett_copy.db`.
2. Open the `sett_copy.db` just like any other database in SQLiteStudio.
3. Execute query: `delete from settings where [group] = 'General' and [key] = 'Session';`
4. Quit SQLiteStudio.
5. Copy `sett_copy.db` back onto the `settings3` file, overwriting it.
6. Run SQLiteStudio3. It should have the layout reset now.

**Reply by novreisb** [reply]
Perfect, it worked. Thks
